frontend/packages/console-app/src/components/quick-starts/QuickStartController.tsx

```diff
@@ -28,6 +34,9 @@ const QuickStartController: React.FC<QuickStartControllerProps> = ({ quickStart
     (task, index) => activeQuickStartState[`taskStatus${index}`],
   ) as QuickStartTaskStatus[];
 
+  const ref = React.createRef<HTMLDivElement>();
+  React.useEffect(() => updateContentRef(ref?.current));
```

Comment:
```suggestion
```

This approach is incorrect because an update to the `ref` doesn't cause a re-render and therefore the effect won't get run unless something else triggers the re-render. You need to use a function as the ref.

But with my other suggestions, you won't need this.